src/build.js

```diff
@@ -92,54 +96,57 @@ async function closeBundleStream (stream, bundleString) {
  * - makes all direct calls to eval indirect
  * - wraps original bundle in anonymous function
  * - handles certain Babel-related edge cases
- * 
+ *
  * @param {string} bundleString - The bundle string
  * @returns {string} - The postprocessed bundle string
  */
-function postProcess (bundleString) {
+function postProcess(bundleString) {
 
   if (typeof bundleString !== 'string') {
-    return null
+    return null;
   }
 
-  bundleString = bundleString.trim()
+  const processedString;
+  processedString = bundleString.trim();
```

Comment:
Let's do this instead:
```suggestion
  let processedString = bundleString.trim();
```